### PR TITLE
TRIVIAL: Update changelog

### PR DESCRIPTION
**Pull request by pbenes**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/1979/ : SUCCESS in 7s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/1454/ : SUCCESS in 7m 08s

**Comment by gdgate** [comment]
Build in pipeline check aborted.

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,22 @@
...
+
+- Use the upgraded GoodStrap library, which now uses the upgraded jQuery library ([commit](TBD)).
+
+## 6.0.1
+
+November 6, 2018
+
+### Changed
+
+- Pivot table no longer shows "is-beta" warning ([commit](https://github.com/gooddata/gooddata-react-components/commit/37d6574d1df0091475a2055096ade7ab5a9875eb)).
```
Comment (evgeniabogranskaya) [inline CHANGELOG.md]: Pivot tables no longer show the "is-beta" warning

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/1981/ : SUCCESS in 6s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/1456/ : SUCCESS in 3m 50s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/1982/ : SUCCESS in 9s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/1457/ : SUCCESS in 1m 53s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/1983/ : SUCCESS in 11s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/1459/ : SUCCESS in 2m 31s

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/458/ : SUCCESS in 31s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/1462/ : SUCCESS in 3m 47s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/807/ : SUCCESS in 12m 36s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/394/ : SUCCESS in 20m 43s

**Pull request merged**